src/Transition/CreateNewIDFUsingRulesV8_7_0.f90

```diff
@@ -427,7 +425,7 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                    OutArgs(35) = 'MutuallyExclusive'
                  END IF
 
-              CASE('AIRFLOWNETWORK:DISTRIBUTION:COMPONENT:DUCT')
+             CASE('AIRFLOWNETWORK:DISTRIBUTION:COMPONENT:DUCT')
```

Comment:
@JasonGlazer These transition changes need to move to new files anyway now, which should leave the 8.6 to 8.7 files untouched.